src/viam/sdk/components/private/board_server.cpp

```diff
@@ -103,8 +103,11 @@ ::grpc::Status BoardServer::ReadAnalogReader(
         extra = struct_to_map(request->extra());
     }
 
-    const Board::analog_value result = board->read_analog(request->analog_reader_name(), extra);
-    response->set_value(result);
+    const Board::analog_response result = board->read_analog(request->analog_reader_name(), extra);
```

Comment:
I named the type `analog_response`, but named all the variables `result`. Should the type be `analog_result` instead? I'm unsure...